R/getExperimentCrossAssociation.R

```diff
@@ -34,15 +34,15 @@
 #'   (Please use \code{assay_name2} instead. At some point \code{abund_values2}
 #'   will be disabled.)
 #' 
-#' @param altExp1 A single numeric or character value specifying alternative experiment
+#' @param altexp1 A single numeric or character value specifying alternative experiment
 #'   from the altExp of experiment 1. If NULL, then the experiment is itself 
 #'   and altExp option is disabled. 
-#'   (By default: \code{altExp1 = NULL})
+#'   (By default: \code{altexp1 = NULL})
 #'   
-#' @param altExp2 A single numeric or character value specifying alternative experiment
+#' @param altexp2 A single numeric or character value specifying alternative experiment
```

Comment:
Do we need 'altExp1' alias? This option was not in release version, and I think it is not that commonly used yet

Reply:
ok to skip if it is not needed